src/librustc/traits/select.rs

```diff
@@ -3066,11 +3095,26 @@ impl<'cx, 'gcx, 'tcx> SelectionContext<'cx, 'gcx, 'tcx> {
         })
     }
 
+    fn confirm_trait_alias_candidate(
+        &mut self,
+        obligation: &TraitObligation<'tcx>,
+    ) -> VtableObjectData<'tcx, PredicateObligation<'tcx>> {
+        debug!("confirm_trait_alias_candidate({:?})", obligation);
+
+        todo
+
+        VtableTraitAliasData {
+            alias_def_id: todo,
+            substs: substs.value,
+            nested: impl_obligations,
+        }
+    }
+
     fn confirm_generator_candidate(
         &mut self,
         obligation: &TraitObligation<'tcx>,
     ) -> Result<VtableGeneratorData<'tcx, PredicateObligation<'tcx>>, SelectionError<'tcx>> {
-        // ok to skip binder because the substs on generator types never
+        // OK to skip binder because the substs on generator types never
```

Comment:
Suggestion: factor these sorts of "nitpick" edits out into a distinct commit... it helps your readers, but it also helps a lot when rebasing.

Reply:
Yep, my bad...

Reply:
No big thing =)